### PR TITLE
add problem link and discuss link in problem file

### PR DESCRIPTION
**Pull request by songyzh**
for conveniently verifying on leetcode and discussing

### CODE REVIEW

File: template.rs [hunk 1]
```diff
@@ -11,6 +11,9 @@ __PROBLEM_DEFAULT_CODE__
 
 // submission codes end
 
+// problem: __PROBLEM_LINK__
+// discuss: __DISCUSS_LINK__
```
Comment (aylei) [inline template.rs]: I'd suggest we keep the informations before the submission codes (closer to the problem description, and make the submission and test case compact)

**Review by aylei — COMMENTED** [review]
The rest LGTM

File: template.rs (hunk 1 above)
Comment (songyzh) [reply template.rs]: sure, fixed https://github.com/aylei/leetcode-rust/pull/26/commits/e62f13afe8c69b69ca8b2949179c02ff706a7d58

**Comment by aylei** [comment]
Thanks!

**Pull request merged**